_posts/2022-06-15-ejb-over-https.adoc

```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: "EJBs over HTTPS: Securing Jakarta Enterprise Beans with mutual TLS authentication"
```

Comment:
Just to prevent confusion with the EJB over HTTP feature, I think it would be good to remove "EJBs over HTTPS" from the title.